Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -144,9 +144,18 @@ public virtual IMixedRealityPointer Pointer
             get { return pointer; }
             set
             {
+                if (ReferenceEquals(pointer?.BaseCursor, this))
```

Comment:
Can you add a comment re: why the BaseCursor property is getting set to null when the references are equal?

Reply:
added